src/badguy/mriceblock.cpp

```diff
@@ -163,6 +163,17 @@ MrIceBlock::collision_badguy(BadGuy& badguy, const CollisionHit& hit)
   return ABORT_MOVE;
 }
 
+void
+MrIceBlock::collision_tile(uint32_t tile_attributes)
```

Comment:
These are some unrelated changes. Can you remove them, please?

Reply:
Yes I know those are changes from my previous commit, I don't know how to use git so I messed up, @maxteufel told me to make a PR and wait...